pkg/gather/common/common.go

```diff
@@ -0,0 +1,35 @@
+package common
```

Comment:
small nitpick: I feel this could be in the `gather` package, no need for a separate package

Reply:
There was a circular dependency since the types are used in gatherer, cluster config and workloads packages and this was the simplest way to avoid it.

Reply:
Let me reopen this one.